### ISSUE TITLE
regression: authentication with provider oidc not supported

### ISSUE THREAD

**Issue by davidB** [clux/kube-rs#424]
before 0.49, kube can authenticate OIDC (see https://github.com/clux/kube-rs/pull/70), but since 0.49/0.50 I got the following error

```
 error: failed to create the kube client
    
    Caused by:
        0: Error loading kubeconfig: Failed exec auth: Authentication with provider oidc not supported
        1: Failed exec auth: Authentication with provider oidc not supported

```

I used the following dependencies

```toml
kube = { version = "0.50", features = ["oauth", "gzip"]}
k8s-openapi = { version = "0.11.0", features = ["v1_16"], default-features = false }
```

**Comment by davidB** [comment]
The issue seems to be fixed on master (but present on tag 0.50.0)

**Comment by davidB** [comment]
see https://github.com/clux/kube-rs/pull/419

**Comment by clux** [comment]
I'll make a release for it later tonight.

**Comment by clux** [comment]
Released in 0.50.1

**Comment by davidB** [comment]
Thanks (also for all the work on kube-rs)
I'm releasing 0.12.1 of kubectl-view-allocations with kube 0.50.1
